src/System.Private.CoreLib/shared/System/Runtime/Serialization/SerializationInfo.cs

```diff
@@ -23,6 +26,125 @@ public sealed class SerializationInfo
         private string _rootTypeName;
         private string _rootTypeAssemblyName;
         private Type _rootType;
+        private readonly static DeserializationToken s_emptyToken = new DeserializationToken(null);
```

Comment:
Should this instead be lazily initialized?  Seems like the 99.999% case is that it'll never be needed.

Reply:
This should not be cached at all. `DeserializationToken` is a struct - it is de-optimization to cache it in a static like this.

Reply:
Good catch... yes this is from when it was a class. Got rid of the caching now that it's a struct.